CubeLineMoment.py

```diff
@@ -685,11 +693,31 @@ def main():
         params['width_map_scaling'] = parse_floatlist(params['width_map_scaling'])
     if 'width_cut_scaling' in params and hasattr(params['width_cut_scaling'], 'split'):
         params['width_cut_scaling'] = parse_floatlist(params['width_cut_scaling'])
-    params['my_line_list'] = u.Quantity(parse_floatlist(params['my_line_list'].split(", "))), u.GHz)
+    params['my_line_list'] = u.Quantity(parse_floatlist(params['my_line_list'].split(", ")), u.GHz)
     params['my_line_widths'] = u.Quantity(parse_floatlist(params['my_line_widths']), u.km/u.s)
     params['my_line_names'] = params['my_line_names'].split(", ")
     if 'sample_pixel' in params:
-        params['sample_pixel'] = ast.literal_eval(params['sample_pixel'])
+        params['sample_pixel'] != None
+#        params['sample_pixel'] = ast.literal_eval(params['sample_pixel'])
+        # Check to make sure that sample pixel regions file exists.  Open it if
+        #  it does exist, and exit script if it does not exist.
+        if os.path.isfile(params['sample_pixel']) == True:
```

Comment:
`== True` is redundant

Reply:
Thanks.
`#        params['sample_pixel'] = ast.literal_eval(params['sample_pixel'])
        # Check to make sure that sample pixel regions file exists.  Open it if
        #  it does exist, and exit script if it does not exist.
        if os.path.isfile(params['sample_pixel']):`